source/styleEditor.js

```diff
@@ -1,3 +1,15 @@
 $(document).ready(function(){
-  // your code goes here.
+
+  function changeStyle(selector, property, value) {
+    $(selector).css(property, value);
+  }
+
+  $('#style_editor').submit(function() {
+    event.preventDefault();
+    let selector = $('#style_editor :input')[0].value
```

Comment:
Is there a more readable way we could grab the values from the forms? Adding an id to the html could be an option or using `$('input[name="selector"]').val()`